client/src/components/Admin/UsersPage.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { PropTypes } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import UsersList from './UsersList';
+import * as actions from './UsersActions';
+
+class UsersPage extends React.Component {
+  constructor(props) {
```

Comment:
If you are not setting state or binding methods in the constructor, you don't need to write the constructor block